### PR TITLE
[CopyFilesV2] Migrated to Node10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: CopyFilesV2

**Description**: Migrated to Node10

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/CopyFilesV2/Tests/L0cleansIfSpecified.ts [hunk 1]
```diff
@@ -41,9 +41,9 @@ runner.registerMockExport('stats', (itemPath: string) => {
     }
 });
 let origReaddirSync = fs.readdirSync;
-fs.readdirSync = (p: string | Buffer) => {
+fs.readdirSync = (p) => {
     console.log('HERE path ' + p);
-    let result: string[];
+    let result;
```
Comment (matskan) [inline Tasks/CopyFilesV2/Tests/L0cleansIfSpecified.ts:46]: Suppose, you can delete that variable at all, and use `return` in `if-else` construction

**Review by anatolybolshakov — APPROVED** [review]

**Review by EzzhevNikita — APPROVED** [review]

**Comment by EzzhevNikita** [comment]
Tested these changes manually found no issues: [pipeline run](https://dev.azure.com/v-niezz-testorg/TestProject/_build/results?buildId=2264&view=results)

**Pull request merged**